packages/typespec-python/src/emitter.ts

```diff
@@ -102,6 +97,8 @@ export async function $onEmit(context: EmitContext<PythonEmitterOptions>) {
 
     const sdkContext = createSdkContext(context);
     const clients = listClients(sdkContext);
+    sdkContext.arm = sdkContext.arm === undefined ? clients.length > 0 && clients[0].arm : sdkContext.arm;
```

Comment:
not sure why `sdkContext.arm` is undefined so add logic to make sure it has specific boolean value